website/blog/2020-09/_index.md

```diff
@@ -0,0 +1,80 @@
+The Gardener API server is now forbidding such requests if the seeds are not completely empty yet.
+
+### Logging improvements for Loki (multiple PRs)
+
+After we released our large logging stack refactoring (from EFK to Loki) with [Gardener v1.8](https://gardener.cloud/blog/2020-08/00/), we are continuing to work on reliability, quality and user feedback in general.
+We aren't done yet, though, Gardener v1.10 includes a bunch of improvements which will help to graduate the `Logging` feature gate to beta and GA, eventually.
+
+## Notable Changes in v1.9
+
+The [v1.9 release](https://github.com/gardener/gardener/releases/tag/v1.9.0) contained tons of small improvements and adjustments in various areas of the code base and a little less new major features.
+However, we don't want to miss the opportunity to highlight a few of them:
+
+### CRI validation in `CloudProfile`s ([gardener/gardener#2137](https://github.com/gardener/gardener/pull/2137))
+
+A couple of releases back we have firstly introduced support for `containerd` and the `ContainerRuntime` extension API.
+The supported container runtimes are operating system specific, and until now it wasn't easily possible for end-users to figure out whether they can enable `containerd` or other `ContainerRuntime` extensions for their shoots.
+With this change, Gardener administrators/operators can now provide this information in the `.spec.machineImages` section in the `CloudProfile` resource.
+This also allows for enhanced validation and prevents misconfigurations.
+
+### New shoot event controller ([gardener/gardener#2649](https://github.com/gardener/gardener/pull/2649))
+
+The shoot controllers in both the `gardener-controller-manager` as well as `gardenlet` are firing several `Event`s for some important operations (e.g., automated hibernation/wake-up due to hibernation schedule, automated Kubernetes/machine image version update during maintenance, etc.).
+Earlier, the only way to prolong the lifetime of these events was to modify the `--event-ttl` command line parameter of the garden cluster's `kube-apiserver`.
+This came with the disadvantage that *all* events are kept for a longer time (not only those related to `Shoot`s that an operator is usually interested in and ideally wants to store for a couple of days).
+The new shoot event controller allows to achieve this by deleting non-shoot events.
+This helps operators and end-users to better understand which changes were applied to their shoots by Gardener.
+
+### Early deployment of the logging stack for new shoots ([gardener/gardener#2750](https://github.com/gardener/gardener/pull/2750))
+
+Since the first introduction of the `Logging` feature gate two years back the logging stack was only deployed at the very end of the shoot creation.
+This had the disadvantage that control plane pod logs aren't kept in case the shoot creation flow is interrupted before the logging stack can be deployed.
```

Comment:
```suggestion
This had the disadvantage that control plane pod logs were not kept in case the shoot creation flow was interrupted before the logging stack could be deployed.
```